sandbox/HealthSandbox/Host.cs

```diff
@@ -76,6 +76,7 @@ private static void Init()
                 .HealthChecks.AddHttpGetCheck("invalid http", new Uri("https://invalid-asdfadsf.com/"), 3, TimeSpan.FromMilliseconds(100), TimeSpan.FromSeconds(1))
                 .HealthChecks.AddHttpGetCheck("github", new Uri("https://github.com/"), retries: 3, delayBetweenRetries: TimeSpan.FromMilliseconds(100), timeoutPerRequest: TimeSpan.FromSeconds(5))
                 .HealthChecks.AddHttpGetCheck("google", new Uri("https://google.com/"), TimeSpan.FromSeconds(1))
+                 // TODO: add SQL Health-Check for sandbox (how?)
```

Comment:
Project not referenced? I'll look at the todo. Thanks for the PR

Reply:
Yes, the project is not referenced for two reasons:
1. unit-tests were not implemented because we need to stub somehow calling SQL server.
2. using SqlChecker in the sandbox wasn't added - otherwise, a user needs a setup DB and provide in configuration (or leave it as is - in this case, SQL check will always be Unhealthy). 

So, seems it should be a decision from your side - use always unhealthy SQL in / add SQL connection from config / suppress warning / etc.